package.json

```diff
@@ -23,7 +23,7 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "set PORT=3001 && react-scripts start",
+    "start": "PORT=3001 react-scripts start",
```

Comment:
Great job so far @enoqueJonas . However, this code is creating an error about the port number. You can fix this issue by using `"start": "set PORT=3001 && react-scripts start",` instead  👍 . This will help ensure your app starts on the correct port.